src/System.Net.Http/src/netcore50/System/Net/HttpClientHandler.cs

```diff
@@ -167,99 +168,48 @@ public bool PreAuthenticate
 
         public bool UseDefaultCredentials
         {
-            get { return Credentials == null; }
+            get { return false; }
```

Comment:
Enterprise Auth is not the only thing that determines if default credentials are used. It also depends on the endpoint itself and whether or not it is listed in the intranet zones.